src/NuGet.Core/NuGet.Commands/PackCommandRunner.cs

```diff
@@ -390,6 +392,70 @@ private static void LoadProjectJsonFile(PackageBuilder builder, string path, str
                 builder.AddFiles(basePath, include.Value, include.Key);
             }
 
+            if (spec.PackOptions != null)
+            {
+                PackCommandRunner runner = new PackCommandRunner(new PackArgs() { BasePath = basePath }, null);
+                if (spec.PackOptions.IncludeExcludeFiles != null)
+                {
+                    string fullExclude;
+                    string filesExclude;
+                    CalculateExcludes(spec.PackOptions.IncludeExcludeFiles, out fullExclude, out filesExclude);
+
+                    if (spec.PackOptions.IncludeExcludeFiles.Include != null)
+                    {
+                        foreach (var include in spec.PackOptions.IncludeExcludeFiles.Include)
+                        {
+                            builder.AddFiles(basePath, include, string.Empty, fullExclude);
+                        }
+                    }
+
+                    if (spec.PackOptions.IncludeExcludeFiles.IncludeFiles != null)
+                    {
+                        foreach (var includeFile in spec.PackOptions.IncludeExcludeFiles.IncludeFiles)
+                        {
+                            var resolvedPath = runner.ResolvePath(new PhysicalPackageFile() { SourcePath = includeFile });
```

Comment:
Ah, yes.  I'd meant to come back to that to clean it up.  I created a static method and switched the other to call the static.